src/lister_haxe/test/testcaseextender.hx

```diff
@@ -0,0 +1,14 @@
+package test;
+
+import haxe.unit.TestCase;
+
+class TestCaseExtender {
+    static public function assertThrows(t:TestCase, fn : Void -> Void) {
```

Comment:
You declare extension methods as a static function with a first param of the type to be extended.  Presumably this is also callable via `TestCaseExtender.assertThrows(t, fn)`... you know, if you wanted to.  The `StringTools` stuff works that way in `ihx`, and at first I didn't realize it was actually doing class extension.